app/graphql/types/query_types/survey_comment.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Types
+  class QueryTypes::SurveyComment < Types::BaseObject
+    field :id, ID, null: true
+    field :survey_poll_id, ID, null: true
+    field :message, String, null: true
+    field :amount, Float, null: true
```

Comment:
`amount` kommt wohl durch copy&paste von `QueryTypes::PriceType` und wird hier nicht benötigt.
dafür allerdings ein `createdAt`. kümmere mich drum.

Reply:
done in 33262702ade72cf2c87c8d2a80487512ebfa2d9a